docs/remote_development.md

```diff
@@ -94,6 +94,10 @@ PRIVATE_KEY_ADMIN=PK1
 ...
 ```
 
+> **Note: The .env file in the aggregator needs to be updated everytime the network changes from local to remote and vice-versa.**
+
+> **.env.example** for **remote**
```

Comment:
Same comment as above, could remove this line. 

I can see that there is no code example here for copying .env.example. Could you add this snippet above like this?

___
_The [aggregator](../aggregator/) is a service that accepts transaction bundles (including those that contain a single transaction) and submits aggregations of these bundles to L2.

```sh
cd ./aggregator
cp .env.example .env
```


Update the following values in `./aggregator/.env`
___